plugin/trino-google-sheets/pom.xml

```diff
@@ -72,7 +72,7 @@
         <dependency>
             <groupId>com.google.apis</groupId>
             <artifactId>google-api-services-sheets</artifactId>
-            <version>v4-rev516-1.23.0</version>
+            <version>v4-rev612-1.25.0</version>
```

Comment:
CI failure.

```
2022-12-20T22:51:32.1433111Z Failed while enforcing RequireUpperBoundDeps. The error(s) are [
2022-12-20T22:51:32.1435074Z Require upper bound dependencies error for com.google.api-client:google-api-client:1.23.0 paths to dependency are:
2022-12-20T22:51:32.1435704Z +-io.trino:trino-google-sheets:404-SNAPSHOT
2022-12-20T22:51:32.1436175Z   +-com.google.api-client:google-api-client:1.23.0
2022-12-20T22:51:32.1436473Z and
2022-12-20T22:51:32.1436837Z +-io.trino:trino-google-sheets:404-SNAPSHOT
2022-12-20T22:51:32.1437349Z   +-com.google.apis:google-api-services-sheets:v4-rev612-1.25.0
2022-12-20T22:51:32.1437860Z     +-com.google.api-client:google-api-client:1.25.0
2022-12-20T22:51:32.1439483Z ]
```
I think you might require to upgrade `google-api-client` as well?